presentation/feedprofile/src/main/java/com/hilingual/presentation/feedprofile/component/FeedProfileInfo.kt

```diff
@@ -46,106 +48,144 @@ internal fun FeedProfileInfo(
-                Text(
-                    text = "${streak}일 연속 작성중",
-                    style = HilingualTheme.typography.bodyM14,
-                    color = if (streak > 0) HilingualTheme.colors.hilingualOrange else HilingualTheme.colors.gray400
-                )
+                        .noRippleClickable(onClick = onFollowTypeClick)
+                        .padding(bottom = 8.dp)
+                ) {
+                    Text(
+                        text = "팔로워",
+                        style = HilingualTheme.typography.captionR14,
+                        color = HilingualTheme.colors.gray400,
+                        modifier = Modifier.padding(end = 2.dp)
+                    )
+                    Text(
+                        text = "$follower",
+                        style = HilingualTheme.typography.bodyB14,
+                        color = HilingualTheme.colors.gray400,
+                        modifier = Modifier.padding(end = 8.dp)
+                    )
+                    Text(
+                        text = "팔로잉",
+                        style = HilingualTheme.typography.captionR14,
+                        color = HilingualTheme.colors.gray400,
+                        modifier = Modifier.padding(end = 2.dp)
+                    )
+                    Text(
+                        text = "$following",
+                        style = HilingualTheme.typography.bodyB14,
+                        color = HilingualTheme.colors.gray400
+                    )
+                }
+                Row(
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Icon(
+                        imageVector = ImageVector.vectorResource(id = R.drawable.ic_fire_16),
+                        contentDescription = null,
+                        tint = HilingualTheme.colors.hilingualOrange,
+                        modifier = Modifier
+                            .padding(end = 1.dp)
+                            .size(16.dp)
+                    )
+                    Text(
+                        text = "${streak}일 연속 작성중",
+                        style = HilingualTheme.typography.bodyM14,
+                        color = if (streak > 0) HilingualTheme.colors.hilingualOrange else HilingualTheme.colors.gray400
+                    )
+                }
             }
         }
+
+        if (!isMine) {
+            FeedUserActionButton(
+                isFilled = isFollowing,
+                buttonText = when {
+                    isBlock -> "차단 해제"
+                    isFollowing -> "팔로잉"
+                    isFollowed -> "맞팔로우"
+                    else -> "팔로우"
+                },
+                onClick = onActionButtonClick,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 20.dp, bottom = 8.dp)
+
+            )
+        } else {
+            Spacer(modifier = Modifier.height(20.dp))
+        }
```

Comment:
```suggestion
        Spacer(modifier = Modifier.height(20.dp))
        
        if (!isMine) {
            FeedUserActionButton(
                isFilled = isFollowing,
                buttonText = when {
                    isBlock -> "차단 해제"
                    isFollowing -> "팔로잉"
                    isFollowed -> "맞팔로우"
                    else -> "팔로우"
                },
                onClick = onActionButtonClick,
                modifier = Modifier
                    .fillMaxWidth()
                    .padding(bottom = 8.dp)
            )
        }
```
P3: 정말 사소한 내용이긴 한데 isMine 여부와 관계 없이 20.dp 패딩이 들어가있으니까 이 부분은 if문 밖으로 빼면 더 가독성이 좋아지지 않을까 싶어요! Spacer는 그대로 넣고 isMine==false 일 때만 FeedUserActionButton를 보여주는 방식으로!!! 요건 취향차이니까 이런 방식도 있구나 하고 넘어가셔도 좋습니다ㅎㅎ